Binaries/DafnyPrelude.bpl

```diff
@@ -917,7 +917,7 @@ axiom (forall<T> s: Seq T :: { Seq#Length(s) }
   );
 
 // The empty sequence $Is any type
-axiom (forall<T> t: Ty :: {$Is(Seq#Empty(): Seq T, t)} $Is(Seq#Empty(): Seq T, t));
+//axiom (forall<T> t: Ty :: {$Is(Seq#Empty(): Seq T, t)} $Is(Seq#Empty(): Seq T, TSeq(t)));
```

Comment:
```suggestion
//axiom (forall<T> t: Ty :: {$Is(Seq#Empty(): Seq T, TSeq(t))} $Is(Seq#Empty(): Seq T, TSeq(t)));
```

Reply:
OK - I wonder if this would have made a difference in the results (when uncommented)